build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -0,0 +1,11 @@
+# Provide the flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp.
+# If no L4 transport proto is given, we consider tcp as default.
+# Defaults to "".
```

Comment:
nit: don't think this line is needed

Reply:
Sorry, do you mean these description lines are not needed for parameters in flow-aggregator.conf?

Reply:
yes, I don't believe we typically include them in the .conf file, since the default value is obvious from the commented-out line

Reply:
Got it, thanks.